pom.xml

```diff
@@ -321,6 +328,41 @@
                             </execution>
                         </executions>
                     </plugin>
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-checkstyle-plugin</artifactId>
+                        <executions>
+                            <execution>
+                                <id>check-google-checks</id>
+                                <phase>verify</phase>
+                                <goals>
+                                    <goal>check</goal>
+                                </goals>
+                                <configuration>
+                                    <configLocation>google_checks.xml</configLocation>
```

Comment:
Should this be part of this PR, or is it somehow built into the plugin?

Reply:
IIRC `google_checks.xml` is either part of the plug-in or checkstyle by default. So doesn't need to be included explicitly.